File: API/Entities/Review.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    public class Review
    {
        public int Id {get;set;}
        public int ProductId{get;set;}
        public Product Product{get;set;}
        public string BuyerId {get;set;}
        public string Comment {get;set;}
        public int Star {get;set;}
        public string ImageReview { get; set; }= string.Empty;
    }
}

File: my-app/src/app/models/Review.ts
export interface Review {
    id:          number;
    productId:   number;
    product:     Product;
    buyerId:     string;
    comment:     string;
    star:        number;
    imageReview: string;
    orderDate:   Date;
}

export interface Product {
    id:              number;
    name:            string;
    imageUrl:        string;
    description:     string;
    price:           number;
    quantityInStock: number;
    categoryId:      number;
    category:        Category;
}

export interface Category {
    id:    number;
    brand: string;
    type:  string;
}
